src/api.js

```diff
@@ -0,0 +1,28 @@
+var url = 'https://en.wikipedia.org/w/api.php';
+
+var params = {
+  action: 'query',
+  generator: 'geosearch',
+  prop: 'coordinates|pageimages',
+  ggscoord: '37.7891838|-122.4033522',
+  format: 'json',
+};
+
+url = url + '?origin=*';
+Object.keys(params).forEach(function (key) {
```

Comment:
It doesn't change the functionality, but I'm inclined to be a stickler for ES6-style functions mainly because not using them could raise questions from someone reviewing a take home test in an interview.

So, this would change to the following:

    Object.keys(params).forEach(key => {
        url += '&' + key + '=' + params[key];
    });